geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -22,7 +22,17 @@ <h1 class="sr-only">
     {% if document.has_transcription %}
         <div class="transcription">
```

Comment:
Could add a class to the div when there's no image to handle in CSS:

```suggestion
        <div class="transcription{% if not document.has_image %} no-image{% endif %}">
```